arch/arm/mach-imx/syscounter.c

```diff
@@ -55,7 +55,7 @@ static inline unsigned long long us_to_tick(unsigned long long usec)
 	return usec;
 }
 
-#ifndef CONFIG_SKIP_LOWLEVEL_INIT
+#if !defined(CONFIG_SKIP_LOWLEVEL_INIT) || defined(CONFIG_SPL_BUILD)
```

Comment:
Not entirely sure if this will have some sort of spill over into other IMX platforms building SPL